docs/src/index.md

```diff
@@ -1,37 +1,41 @@
 # DensityInterface.jl
 
-This package defines an interface for mathematical/statistical densities in Julia. The interface comprises the functions [`isdensitytype`](@ref),  [`logdensityof`](@ref) and [`logfuncdensity`](@ref).
+```@docs
+DensityInterface
+```
+
+This package defines an interface for mathematical/statistical densities and objects associated with a density in Julia. The interface comprises the functions [`hasdensity`](@ref),  [`logdensityof`](@ref)/[`densityof`](@ref) and [`logfuncdensity`](@ref).
 
-A the following methods must be provided to make a type (e.g. `SomeDensity`) compatible with the interface:
+The following methods must be provided to make a type (e.g. `SomeDensity`) compatible with the interface:
 
 ```julia
```

Comment:
Maybe also make this executable and add a hidden definition of a simple `SomeDensity` and `log_of_d_at_x`?

Reply:
Will do.

Reply:
Done.